lib/facade/MonitoringFacade.ts

```diff
@@ -209,15 +216,23 @@ export class MonitoringFacade extends MonitoringScope {
   // =======
 
   createdDashboard(): Dashboard | undefined {
-    return this.dashboardFactory?.createdDashboard();
+    const dashboardFactory = this.dashboardFactory as IDashboardFactory;
```

Comment:
My understanding is that `createdDashboard()` is basically a method alias for getting the created 'detail' dashboard. This method is provided by the original `IDashboardFactory` interface which means that it should always work as expected unless someone has chosen to use the new dynamic dashboard functionality and chooses to call this method instead of calling the new `getDashboard(type)` method. This case would be a programmer error, but I can add some safety rails here to let people know their mistake if you would like. 

Let me know what you think of the above and if my understanding may not be correct.

Reply:
I added the requested type checks here.